xinference/model/llm/ggml/ctransformers.py

```diff
@@ -42,10 +42,11 @@
     "starcoder": "starcoder",
     "starchat": "starcoder",
     "falcon": "falcon",
+    "mistral-v0.1": "mistral",
 }
 
 # these two constants subjects to change for future development and ctransformers updates.
-CTRANSFORMERS_SUPPORTED_MODEL = ["starcoder", "gpt-2"]
+CTRANSFORMERS_SUPPORTED_MODEL = ["starcoder", "gpt-2", "mistral-v0.1"]
```

Comment:
`llama.cpp` now also supports Mistral.

I've reviewed the recent activity on `ctransformers`, and it seems the last commit was on 9/10. Given that `llama.cpp` has expanded its support to include non-llama models like falcon and starcoders, `llama.cpp` should be considered as the first choice to run `ggml` or `gguf` models.

What do you think?

Reply:
That makes sense, working on it!